sinon/index.d.ts

```diff
@@ -427,7 +427,7 @@ declare namespace Sinon {
 
     // Utility overridables
     interface SinonStatic {
-        createStubInstance(constructor: any): SinonStub;
+        createStubInstance(constructor: any): any;
```

Comment:
How about ["Mapped Types"](https://www.typescriptlang.org/docs/handbook/advanced-types.html)?
```ts
createStubInstance<T>(constructor: { prototype: T }): { [P in keyof T]: SinonStub };
```
If you do this you will need to add `// TypeScript Version: 2.1` as the last line of the header (line 5).